x11-misc/j4-dmenu-desktop/j4-dmenu-desktop-2.17.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+MY_PV="r${PV}"
+
+DESCRIPTION="A fast desktop replacement for i3-dmenu-desktop"
+HOMEPAGE="https://github.com/enkore/${PN}"
+SRC_URI="https://github.com/enkore/${PN}/archive/r${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="test? ( dev-cpp/catch:0 )"
```

Comment:
CMakeLists.txt still specifies Catch1.x. Is it working with 2.x?

Reply:
Yes. I did run a test and all tests run successfull with dev-cpp/catch:0.

Reply:
With catch 2.3.0 I get
>CMake Error at CMakeLists.txt:53 (message):
>  Catch include directory not found

Reply:
Yeah, I see my mistake, I've forgotten to remove the old slot, so it didnt' have triggered. It's already reverted in last commit.